package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@emotion/core": "^10.0.15",
     "@emotion/styled": "^10.0.15",
+    "@types/react": "16.8.23",
```

Comment:
what's the reasoning for making this an exact dependency over a non-locked dev dependency?

Reply:
Solid point I think it was an oversight because I installed that as a specific dependency to match what was in devDependencies. I'll update.